fastlane/spec/actions_specs/swiftlint_spec.rb

```diff
@@ -101,6 +101,31 @@
         end
       end
 
+      context "when specify path options" do
+        it "adds path option" do
+          path = "spec/fixtures"
+          result = Fastlane::FastFile.new.parse("
+            lane :test do
+              swiftlint(
+                path: '#{path}'
+              )
+            end").runner.execute(:test)
+
+          expect(result).to eq("swiftlint lint --path #{path}")
+        end
+
+        it "adds invalid path option" do
+          path = "/path/to/lint"
+          expect do
+            Fastlane::FastFile.new.parse("lane :test do
+              swiftlint(
+                path: '#{path}'
+              )
+            end").runner.execute(:test)
+          end.to raise_error("Couldn't find path '#{path}'")
```

Comment:
The issue with the failing Windows test is it is putting a `C:` in front of the error so its not matching that "exact" string. Try... `end.to raise_error(/Couldn't find path/)`